app/src/main/java/io/apicurio/registry/rules/RulesServiceImpl.java

```diff
@@ -117,8 +131,22 @@ public void applyRule(String artifactId, long artifactVersion, ArtifactType arti
             throws RuleViolationException {

```

Comment:
While you're in here, can you rename this method to `applyRules` since I believe it applies all rules not just one?  IMO this was not properly named originally.

Reply:
Done 👍